seeds/platypus_finance/platypus_finance_avalanche_c_example_trades.csv

```diff
@@ -0,0 +1,21 @@
+block_time,token_bought_amount,token_sold_amount,tx_hash,evt_index
+2022-05-01 00:56,4994.308573,4994.787342,0xc9cf002b6422ef0a617284537683372c66b92e84e0b28fde8a0cc04a4eef519e,23
+2022-05-03 02:21,21009.805223,21012.319756,0x75dc4b71defb76d9888aabbd8771e8b38ee48fb41d43cc3ecae6fb73b3911c84,20
+2022-05-03 14:57,1199719.208427,1200000,0x93bd89cf8a4d602d5cbc32446e5fb4bf9ed170f5ac72c2dc23294c8f5e1a8a05,26
+2022-05-04 03:08,220.02067369540612,220,0x306818d93ecd131c5e5e40a2293150db9484555d539a45e0512cc28a7041ebfb,45
+2022-05-04 23:30,728.2722352237333,728.227031,0x9e982c5f221d878d5c30291f3b1af3bb4896a0d15ecc305ff4d4e63936ed191e,64
+2022-05-05 20:44,148967.354389,149000,0xd1d05ff16c664884875cf17ded334008fe1005b15103460c76a8979a791d3cc1,13
+2022-05-05 22:06,783.164086,783.2365264361647,0x138e58cdee3bd8ccb09bfaaffe340f84c29688d4a239dab4b94d12e49434d5a9,82
+2022-05-06 07:15,741.001182,741.1522427698851,0x48596630bee61338ac47f100539d58fd783215ceb624d888249818b9c7eade5f,59
+2022-05-07 07:05,1034.338934536323,1034.310287,0x2954327b19870057c067530f4f951014163795afe991f5cd2657649ab8151a88,74
+2022-05-07 17:19,121.298967,121.300365,0x9ac609a5cf6084d4152e835fcf790f45dbdb363bda79afc62ac3dfc5235eea7d,17
+2022-05-07 23:37,2069.426263,2069.863443,0x2a3a710fa23fe85c0153a701167ac8265327e08c4e527e9de3d0727099934f48,8
+2022-05-09 12:36,504890.913495,505000,0xa268e678b9167e2afbff624d6c473e1de4e69e00bbbf15cff30fec754965161e,80
+2022-05-09 12:59,23608.565879,23614.60193014132,0x2747e96f2e6198cbaad9a34257faae3d1282401ec89c4d8e2b8980830417f7e4,26
+2022-05-09 17:27,340.040453,340.06469425688414,0x4587aca823136fd6dbe9ecf9a992074062c49d13ee38fc65a7f547e5647c40d6,34
+2022-05-09 19:00,25017.262697,25023.099847,0x916f2f560b3c13f31aa4139608b067e00ad042bd3bdb197a683e68d77c80aab0,12
+2022-05-09 19:06,289498.193117,289560.3059171445,0xd1e3319eb5ab9cae929d2a18e79facb10309f84a8e38073596df084745c2f2f1,113
+2022-05-09 22:55,7500.368177,7500,0x7fa1daa95f0c034752cb3719b62e7ae3d372945db163bda78e26cdc3a18192c3,6
+2022-05-09 23:09,78918.176151,78938,0x363e3d084f738cb84cc809ca6ec9738bc23ceb606ca13148fe210b0bb098115b,22
+2022-05-09 23:33,44548.677919,44561,0x70aa1e9d8f0698b0b4b118b5296d67a3fb15f380a8ca105ba3c433655ae93aa6,28
+2022-05-10 00:55,1505.784684,1506.0280424835855,0x1bbe2f7773c059f500de2cd6acdee778d04b8c8185d4c69bb20db835feda9b76,24
```

Comment:
Hi @UmerHA , since this is a models under dex.trades, we have [a seed file](https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv) for the trades from all dexs on all chains. You can put these seeds into the file as well.